src/main/java/Duke.java

```diff
@@ -1,10 +1,275 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+
+
 public class Duke {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws DukeException, IllegalArgumentException {
+
+        List<Task> list = new ArrayList<>(); // to store list of inputs
+
+
+        // attempt to open file within same folder as src code, create file if file doesn't exist
+        try {
+            File file = new File("./tasks.txt");
+            if (!file.exists()) {
+                System.out.println("File doesn't exist, creating file");
+                file.createNewFile();
+                System.out.println("File created");
+            } else {
+                System.out.println("File exists");
+                // read from file and add to task list
+            }
+        } catch (IOException e) {
+            System.out.println("Error while opening file");
+        }
+
+        // at this point file will exist, init taskList
+        listInit(list);
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String welcomeMsg = "Hello! I'm\n" + logo + "\nWhat can I do for you?\n";
+        System.out.println(welcomeMsg);
+
+        Scanner sc = new Scanner(System.in);
+        String input = "";
+
+        while (true) {
+
+            try {
+                input = sc.nextLine();
+
+                if (input.equals("bye")) {
```

Comment:
A switch case could be used here instead of if-elses

Reply:
that makes a lot more sense, dont know why i didn't think of that. Thank you!